### PR TITLE
Remove some easy/simple wording

### PR DESCRIPTION
**Pull request by Charlotteis**
Hello :wave: :sparkles:!

I'm going to learn Hapi with this tutorial! First, I wanted to suggest a few removals of the word easy/simple. I could talk about why for hours but the short version is: nothing is ever easy/simple.

Am more than happy to give the long version too :dancer:.

Thanks for writing this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -438,7 +438,7 @@ We have 4 options:
 1. Google - If you are building an "enterprise" or "education" app
 which you know will be used in Google-enabled companies/schools we
 wrote a Hapi plugin: https://github.com/dwyl/hapi-auth-google which
-lets you include Google Login in your app in a few simple steps. The plugin uses the [***Official Google Node.js API Client***](https://github.com/google/google-api-nodejs-client) and is
+lets you include Google Login in your app in a few steps. The plugin uses the [***Official Google Node.js API Client***](https://github.com/google/google-api-nodejs-client) and is
```
Comment (iteles) [inline README.md]: How about in a few 'clear' steps? What we want to convey here is that an effort has been made to make the process as accessible as possible independently of a person's coding experience, so I feel like we need _some_ form of adjective here... Though I might just be being picky.

**Comment by iteles** [comment]
@Charlotteis Thanks for this!! We probably have to do a check of all of our tutorials for this issue.

I particularly liked to see that you're not applying your rule with one large sweeping stroke, but have the flexibility in there when needed (e.g. use of `easier`) :+1: :+1:

**Comment by Charlotteis** [comment]
@iteles good adjectiving, would adjective again. Updated!

I have a bunch of the dwyl tutorials I want to read now, so maybe I'll find some simple/easy words along the way. Thanks for the speedy feedback! :sparkles:

**Comment by iteles** [comment]
Excellent, thanks again for this @Charlotteis :ok_hand: 

We definitely welcome your contributions :blush:

**Comment by nelsonic** [comment]
Thanks for fixing that. :+1:

**Pull request merged**